documentation/modules/ROOT/pages/connectors/jdbc.adoc

```diff
@@ -537,10 +539,40 @@ The {prodname} JDBC sink connector has several configuration properties that you
 Many properties have default values.
 Information about the properties is organized as follows:
 
+* xref:jdbc-connector-properties-generic[]
 * xref:jdbc-connector-properties-connection[]
 * xref:jdbc-connector-properties-runtime[]
 * xref:jdbc-connector-properties-extendable[]
 
```

Comment:
I missed this earlier, but these link lists that appear in the documentation for the other connectors are designed for downstream use (conditionalized as `product`]) , where the specified IDs match anchors applied to the downstream version of the docs. I forget now if and when we're planning to expose the JDBC connector downstream, but if we do, we'll have to make sure that we retain the upstream anchor IDs vs. using the `ModuleID` comment to modify the IDs for downstream. Otherwise the empty link text causes problems.
 Just a comment for now ... no action required.

Reply:
We likely will at some future point, I can't say when @roldanbob, so if it makes less work for later to fix this to be compliant for what we may need, I'm all for it.

Reply:
No issue! I'll merge.